concrete/src/Form/Service/Widget/DateTime.php

```diff
@@ -179,6 +180,17 @@ public function datetime($field, $value = null, $includeActivation = false, $cal
             $stepHours = 1;
         }
 
+        // Create datepicker properties
+        $dpOptions = '';
+        foreach ($datePickerOptions as $key => $value) {
+            if (is_bool($value)) {
```

Comment:
What about using json_encode? It works for booleans, correctly escapes strings, keeps numbers as numbers...